tavern/tomes/process_info/main.eldritch

```diff
@@ -0,0 +1,17 @@
+def process_info(pid):
+    info = process.info(sys.get_pid() if pid == -1 else pid)
+    for key, value in info.items():
+        if key == "cmd":
+            print("cmd_args:\n")
+            for nested_value in value.split(" "):
+                print("\t- {}\n".format(nested_value))
+        elif key == "environ":
+             print("env_variables:\n")
+             for nested_value in value.split(","):
+                print("\t- {}\n".format(nested_value))
+        else:
+            print("{}={}\n".format(key, value))
```

Comment:
Nit could you change the `=` to `: ` for consistency

```
pid=8531
name=golem
cmd_args:
        - /workspaces/realm/implants/target/debug/golem
        - /tmp/test.eldritch
exe=/workspaces/realm/implants/target/debug/golem
env_variables:
        - BROWSER=/vscode/vscode-server/bin/linux-x64/8b3775030ed1a69b13e4f4c628c612102e30a681/bin/helpers/browser.sh
        - CARGO=/usr/local/rustup/toolchains/nightly-2023-09-04-x86_64-unknown-linux-gnu/bin/cargo
        - CARGO_HOME=/usr/local/cargo
        - CARGO_MANIFEST_DIR=/workspaces/realm/implants/golem
        - CARGO_PKG_AUTHORS=
 ```

Reply:
should be fixed